wolfcrypt/src/cmac.c

```diff
@@ -216,6 +216,7 @@ int wc_CmacFinal(Cmac* cmac, byte* out, word32* outSz)
 {
     int ret;
     const byte* subKey;
+    word32 remainder = 0;
```

Comment:
You may get a set but not used with 0 init. Recommend removing the `=0`. This can also move into the else section. Variable declarations must be at top of function or brace section.

Reply:
I've removed the init `= 0` and left the declaration at the top of the function.